esmvaltool/cmorizers/data/cmor_config/NOAA-MBL-CH4.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: NOAA-MBL-CH4
+  version: '1.0'
+  tier: 2
+  modeling_realm: atmos
+  project_id: OBS
```

Comment:
It might be better to use "OBS6" for this dataset. There are not super hard rules on when to use "OBS6" or "OBS", but we tend to use "OBS6" for the newer datasets. It is normally an indicator that the dataset is compliant with CMIP6 (and not CMIP5) variables. Since this is a derived variable anyway, it does not matter so much. However, when the dataset is used for comparisons with CMIP6, it would make it more consistent for the recipe to use "OBS6". Did you have a very specific reason to use "OBS" here?